### DISCUSSION TITLE
Does it have a Designer ?

### DISCUSSION THREAD

**Discussion by hichammounadi** [thomasduft/microwf#26] [General]
First, thanks for sharing,
I wanna ask if it has a designer so that I can design my own workflow by dragging and dropping activities, choosing actors like the workflowEngine.net, or it's just hard coding the workflow?

**Comment by thomasduft** [comment]
Hi 

The answer to your question is no. That's why I wrote "A simple finite state machine (FSM) with workflow character where you define your workflows in code" in the description. 

In my experience those "Low Code" real workflow engines that ship with a designer never really turned out working nicely at customers. Sooner or later some admin thinks he knows what he is doing and then it ends in an "Hot mission incident" where he is calling you in the middle of the night ;-). 
My past thought me that you should first understand the customers workflow and then you need a developer anyway who implements it - in code!

Hope this helps you in some way. Have a nice day.

Thomas
